layer.yaml

```diff
@@ -1,31 +1,59 @@
-includes: ['layer:hadoop-client', 'interface:spark', 'interface:benchmark']
+includes:
+- layer:hadoop-client
+- interface:spark
+- interface:benchmark
 options:
-  hadoop-client:
+  apache-spark:
     packages:
-      - 'ipython'
-      - 'libgfortran3'
-      - 'scala'
+    - ipython
+    - libgfortran3
+    - scala
     dirs:
       spark:
-        path: '/usr/lib/spark'
+        path: /usr/lib/spark
       spark_conf:
-        path: '/etc/spark/conf'
+        path: /etc/spark/conf
       spark_logs:
-        path: '/var/log/spark'
-        owner: 'ubuntu'
-        group: 'hadoop'
+        path: /var/log/spark
+        owner: ubuntu
+        group: hadoop
       spark_work:
-        path: '/var/lib/spark/work'
-        owner: 'ubuntu'
-        group: 'hadoop'
+        path: /var/lib/spark/work
+        owner: ubuntu
+        group: hadoop
     ports:
-      # Ports that need to be exposed, overridden, or manually specified.
-      # Only expose ports serving a UI or external API (i.e., namenode and
-      # resourcemanager).  Communication among units within the cluster does
-      # not need ports to be explicitly opened.
       spark-history:
         port: 18080
-        exposed_on: 'spark'
+        exposed_on: spark
       spark-webui:
         port: 8080
-        exposed_on: 'spark'
+        exposed_on: spark
+    users: []
+    groups: []
+  livy:
+    users:
+      hue:
+        groups: [hadoop]
+    dirs:
+      livy:
+        path: /usr/lib/livy
+        owner: hue
+        group: hadoop
+      livy_conf:
+        path: /etc/livy/conf
+        owner: hue
+        group: hadoop
+      livy_logs:
+        path: /var/log/livy
+        owner: hue
+        group: hadoop
+    ports:
+      livy:
+        port: 8998
+        exposed_on: spark
+    groups: []
+  basic:
+    use_venv: false
+    packages: []
+    include_system_packages: false
+is: apache-spark
```

Comment:
The `is` key is an artifact of the build process and shouldn't be set by layers.  (Though, I doubt it hurts anything.)